owncloud/phoenix
# Start to support more languages: es, cs, fr, it

## Description
Up to now only de was supported as translation.
This PR adds support for es, cz, fr and it

## How Has This Been Tested?
- switch the language in your browser to es
- reload phoenix
- see texts translated in Spnish

## Screenshots (if appropriate):
![Screenshot from 2019-04-12 14-52-24](https://user-images.githubusercontent.com/1005065/56038598-a46f0d00-5d32-11e9-83dc-c643d4fe313e.png)

![Screenshot from 2019-04-12 14-49-18](https://user-images.githubusercontent.com/1005065/56038451-48a48400-5d32-11e9-817c-62e8d5b10bcd.png)


## Types of changes

## Checklist:



## Open tasks:
